### ISSUE TITLE
hl_ver missing from compiler flags table

### ISSUE THREAD

**Issue by uvtc** [HaxeFoundation/HaxeManual#413]
I don't see `hl_ver` in the table at <https://staging.haxe.org/manual/compiler-usage-flags.html>. Is it supposed to be in there?

@Gama11 pointed out use of this flag at <https://community.haxe.org/t/haxe-4-0-0-rc-3-is-released/1808/4>.

**Comment by Aurel300** [comment]
Yeah, there are still some defines in the compiler that are not declared in the JSON files (from which the tables are generated). Also applies to some static analyser defines. We can keep this open for defines missing in general.
